skymp5-server/cpp/lib_espm/espm.cpp

```diff
@@ -282,8 +258,9 @@ class espm::RecordHeaderAccess
         try {
           const auto inSize = rec->GetFieldsSizeSum() - sizeof(uint32_t);
           ZlibDecompress(ptr, inSize, out->data(), out->size());
-        } catch (std ::exception& e) {
-          assert(0 && "ZlibDecompress has thrown an error");
+        } catch (const std::exception& e) {
```

Comment:
Let's remove try/catch at all and let `ZlibDecompress` just throw its exceptions.